GitHubConfiguration.ps1

```diff
@@ -146,6 +146,11 @@ function Set-GitHubConfiguration
         created/updated and the specified configuration changes will only remain in memory/effect
         for the duration of this PowerShell session.
 
+    .PARAMETER GitHubBaseUrl
```

Comment:
I can. Historically when I have added functionality to systems, preserving backwards compatibility trumped formatting. In this case moving the parameter up in the parameter list would break compatibility in the event that the function is called without explicit argument names (`Test-Path "/some/dir"` instead of `Test-Path -Path "/some/dir"`.)

If you're okay with breaking backwards compatibility I'm fine with reordering it to alphabetical.

Reply:
A concern on back compat is _quite_ valid.  For this method _specifically_ though, I'm not concerned.  The usage pattern for this method is such that it is _highly_ unlikely that users would ever call this and specify _all_ of the parameters simply by position.  Users are much more likely to either call it multiple times specifying one parameter each time, or to call it a single time and explicitly specify the parameter name with its value.

So, while I _completely_ agree that we should avoid back compat breaking changes to parameter ordering _in general_, I'm _not_ concerned with that problem with _this_ function specifically.

Thanks for raising the concern though.

Reply:
Excellent, I've got some time before bed so expect an updated changeset for review shortly.

Reply:
Reordered according to naming.